### ISSUE TITLE
Wrong predict results on onnx model 1.2 from customvision.ai

### ISSUE THREAD

**Issue by Gaploid** [Microsoft/onnxruntime#487]
**Describe the bug**
The model is trained on Customvision.ai and exported as onnx 1.2 model. I tested images using web-site and onnxruntime and results are not the same. They are completely different. The model using onnxruntime always output only one same class for all images.  

**System information**
- Linux Ubuntu 16.04.4 LTS
- ONNX Runtime version: 0.2.1
- Python version: 3.5

**To Reproduce**
Download the model from here: https://functionaed97c258575.blob.core.windows.net/regions/apex2.onnx
Download test image: https://functionaed97c258575.blob.core.windows.net/regions/out-up15726592.jpg

Execute that script (example based on that code - https://docs.microsoft.com/en-us/python/api/overview/azure/onnx/auto-examples-plot-dl-keras?view=azure-onnx-py )
```
import os
import keras
import tensorflow

import numpy
import onnxruntime as rt
from skimage.transform import resize
from keras.preprocessing.image import array_to_img, img_to_array, load_img
import matplotlib.pyplot as plt

img1 ='dead12556851.jpg'

img = load_img(img1)
ximg = img_to_array(img)

plt.imshow(ximg /255)
plt.axis('off')

ximg224 = resize(ximg /255, (227, 227,3),anti_aliasing=True)
ximg = ximg224[None, :, :, :]
ximg = ximg.astype(numpy.float32)
ximg = numpy.moveaxis(ximg,3,1)
print("new shape:", ximg.shape)

sess = rt.InferenceSession('/home/gaploid/apex2.onnx')

print("The model expects input shape:", sess.get_inputs()[0].shape)
print("image shape:", ximg.shape)
print (sess.get_inputs()[0].name)

input_name = sess.get_inputs()[0].name
res = sess.run(None, {input_name: ximg})
prob = res[0]
print(res)

#output:[array([['none']], dtype=object), [{'none': 1.0, 'knock': 0.0, 'assist': 0.0, 'dead': 0.0, 'eliminate': 0.0}]] 
#i'm always get result like that for all images. If I will test that image with that model on CustomVision.ai I will get correct label result - "dead" with 100% accuracy. 
```

**Expected behavior**
Different output prediction result is at least same as output result on customvision.ai web-site.

**Comment by Gaploid** [comment]
I figured out that onnx model from customvision.ai is waiting in pixels in int and not float and also in BGR format. That change behavior and the model start give different results but these results are almost wrong. Probably I need to create that issues in customvision github repo.

**Comment by snnn** [comment]
Good finding!
Is your problem solved?

**Comment by Gaploid** [comment]
@snnn Yep, everything is working fine now. But CustomVision folks should provide description for exported model cause I`ve spend 2 hours generating different images and test the model to understand in what format the model is waiting input data:)

**Comment by pranavsharma** [comment]
@Gaploid please close this if your issue is resolved. Thx.

**Comment by ellerado** [comment]
Could you please share your script? I'm having the similar issue and i'm not sure what i'm doing wrong. Would really appreciate it. Thanks!

**Comment by prakharter** [comment]
Having the same issue , Pleas advise what did you changed with your image transformation to get the accuracy working , thanks

**Comment by HoaNguyen55** [comment]
I am facing this issue now, is there any recommends to solve this problem ?
